src/components/close-button/CloseButton.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import Button, { ButtonType } from '../button';
+import IconClose from '../../icons/general/IconClose';
+import { bdlGray62 } from '../../styles/variables';
+import './CloseButton.scss';
+
+export interface CloseButtonProps {
+    className?: string;
+    onClick?: Function;
+}
+
+const CloseButton = ({ className, onClick }: CloseButtonProps) => {
```

Comment:
Since this is a new component, can we add the `index`, `stories`, `flow` files similar to: https://github.com/box/box-ui-elements/tree/master/src/components/plain-button